### PR TITLE
[TT-314] More Test Contract Cleanup - Simple(Read/Write)AccessController

### PR DESCRIPTION
**Pull request by tateexon**

### CODE REVIEW

**Review by AnieeG — DISMISSED** [review]

File: integration-tests/contracts/contract_deployer.go [hunk 1]
```diff
@@ -191,14 +192,14 @@ func (e *EthereumContractDeployer) DeployReadAccessController() (ReadAccessContr
...
-		return ethereum.DeploySimpleReadAccessController(auth, backend)
+		return simple_read_access_controller.DeploySimpleReadAccessController(auth, backend)
 	})
 	if err != nil {
 		return nil, err
 	}
 	return &EthereumReadAccessController{
 		client:  e.client,
-		rac:     instance.(*ethereum.SimpleReadAccessController),
+		rac:     instance.(*simple_read_access_controller.SimpleReadAccessController),
```
Comment (AnieeG) [inline integration-tests/contracts/contract_deployer.go:202]: Is it being used in any test currently?

Comment (kalverra) [reply integration-tests/contracts/contract_deployer.go:202]: Same question. We might be able to rip this one out entirely.

Comment (tateexon) [reply integration-tests/contracts/contract_deployer.go:202]: Oh that is a good question, no usages in chainlink at least. Will dig through the other repos.

**Review by AnieeG — APPROVED** [review]

File: integration-tests/contracts/contract_deployer.go (hunk 1 above)
Comment (tateexon) [reply integration-tests/contracts/contract_deployer.go]: Ripped out and cleaned up the PR. Good call out!

**Pull request merged**